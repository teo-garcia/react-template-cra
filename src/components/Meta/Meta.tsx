import { Helmet } from 'react-helmet';
import { MetaProps } from '../../tools/types';

function Meta(props: MetaProps) {
  const { title, description } = props;
  return (
    <Helmet>
      <title>{title}</title>
      <meta name="description" content={description} />
    </Helmet>
  );
}

export { Meta };
