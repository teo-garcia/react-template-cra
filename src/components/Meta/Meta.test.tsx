import { render } from '@testing-library/react';
import { Meta } from './Meta';

describe('<Meta /> tests', function () {
  test('Should match with the component snapshot', function () {
    const { asFragment } = render(
      <Meta title="Ping | Home" description="This is the homepage of Ping" />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
