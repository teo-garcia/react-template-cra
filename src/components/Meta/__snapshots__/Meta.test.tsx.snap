// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Meta /> tests Should match with the component snapshot 1`] = `<DocumentFragment />`;
